examples/Backtesting/botorch_analytical.py

```diff
@@ -111,8 +110,11 @@
 )
 
 # We use the plotting utility to create plots.
+# Note that the path chosen here refers to the current working directory.
```

Comment:
I strongly vote against adding this to the examples, for several reasons. Our main effort should be to **reduce** our overly verbose / non-minimal examples. This brings us in the exact opposite direction!
* We should stop copy-pasting stuff from examples to examples. This is a maintenance nightmare
* The has nothing to do with the examples at all. Yet, it takes most of the written text in the examples
* This is nothing the user should even have to bother about in the first place (see suggestion below)
* It should really not be necessary to broadly explain what `Path(".")` does. Everyone could look that up themselves.

In fact, I suggest this should not be visible to the user in the first place. Why not simply make `Path(".")` as the default of our plotting utility? After all, I thing no one is ever gonna use a different path anyway. That would  keep the example clean and restrict the non-example related code to exactly one line, namely the call of the utility.

Reply:
Will change by making it the default.

Reply:
Made the corresponding change, and made it a keyword-only argument with `Path(".")` as the default. Feel free to resolve if you are happy with this solution.

Reply:
Yes, much nicer, indeed. Curious: why did you make it keyword-only instead of regular?

Reply:
There was no strong reason for that decision. I just thought that it might make it more clear that an actual path would be required. If you see no reason for making it kw-only I can also easily make that a regular argument.

Reply:
OK, I see. In that case, I think a regular argument makes more sense, tbh. Could it be that you are confusing the role of three (mostly independent) properties of signature parameters?
* They can have defaults or not
* The can be "optional" or not (e.g. may take the value `None`), which is different from the above
* They can be positional-only, regular, or keyword-only.

The latter has only partly to do with defaults. You'd typically change it depending on whether you specifically **want** people to use the given name (e.g. since otherwise the context could be lost in the call) or specifically **do not** want to use that name (e.g. when working with "arbitrary" names like "x" and / or to avoid backward incompatibility)

Reply:
Thanks for the explanation :) I wouldn't say that I confuse them, since in my head it made sense to force people to actually use the `path` name, but I fully agree that my argument for making people use it makes not that much sense, and that yours is the better way to think about it.

Reply:
Changed and made that into a regular argument having `Path(".")` as default.